utilities/ride-model-util/src/test/java/com/adobe/ride/utilities/model/test/ModelObjectTest.java

```diff
@@ -106,6 +106,46 @@ public void testStringJsonSchema() throws ModelSearchException {
     Assert.assertTrue(test.containsAll(control));
   }
 
+  @SuppressWarnings("unchecked")
+  @Test(suiteName = "smoke", groups = "integration", enabled = true)
+  public void testSchemaStringStatic() throws ModelSearchException {
+    // Model String
+    String modelString = "";
+    try {
+      InputStream inputStream =
+          ClassLoader.class.getResourceAsStream("/schemas/TestService/simple.json");
+      Charset nullCharset = null; // platform default
+      modelString = IOUtils.toString(inputStream, nullCharset);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    // Presets
+    JSONObject presets = new JSONObject();
+    JSONObject title = new JSONObject();
+    title.put("title", "Gladys Phillips");
+    presets.put("title", title);
+
+    // Targets
+    Set<String> targetNodes = new HashSet<String>();
+    targetNodes.add("/title");
```

Comment:
Thanks for adding this test!  One more final request.  Can you add a few more target nodes here to ensure that things other than preset nodes are added to the instance?

Reply:
Done!